ingestion/src/metadata/clients/databricks_client.py

```diff
@@ -113,3 +115,10 @@ def list_query_history(self, start_date=None, end_date=None) -> List[dict]:
             logger.error(exc)
 
         return query_details
+
+    def is_query_valid(self, row) -> bool:
+        query_text = row.get("query_text")
+        return not (
+            query_text.startswith(QUERY_WITH_DBT)
+            or query_text.startswith(QUERY_WITH_OM_VERSION)
+        )
```

Comment:
If it doesn't apply only to Databricks could we have it in a utils package? What do you think? 
nit: doc on the method

Reply:
this only applies to databricks as of now for other connectors we are filtering via qureries.